### PR TITLE
Bootstrap slider

### PR DESCRIPTION
**Pull request by reiern70-antilia**

### CODE REVIEW

**Comment by reiern70** [comment]
@martin-g 

I think I have covered all the things you mentioned on previous PR. Please let me know if anything else needs any action on my side. Thanks again!

Cheers,

Ernesto

File: bootstrap-extensions/src/main/java/de/agilecoders/wicket/extensions/slider/res/css/bootstrap-slider.css [hunk 1]
```diff
@@ -0,0 +1,255 @@
+/*! =======================================================
+                      VERSION  5.3.7
```
Comment (martin-g) [inline bootstrap-extensions/src/main/java/de/agilecoders/wicket/extensions/slider/res/css/bootstrap-slider.css]: I've noticed that v.6.x is released two days ago.
Do you want the newest version or 5.3.7 is good enough?

**Comment by martin-g** [comment]
Please squash the commits so I can easily merge it back to 6.x.
Thanks a lot!

File: bootstrap-extensions/src/main/java/de/agilecoders/wicket/extensions/slider/res/css/bootstrap-slider.css (hunk 1 above)
Comment (reiern70) [inline bootstrap-extensions/src/main/java/de/agilecoders/wicket/extensions/slider/res/css/bootstrap-slider.css]: I have just upgraded the files: thanks for spotting that

**Comment by reiern70** [comment]
Done! Thanks to you!

**Comment by martin-g** [comment]
Up and running at: http://wb-mgrigorov.rhcloud.com/slider !

**Comment by reiern70** [comment]
Thanks!

Sent from my ASUS

-------- Original Message --------

**Pull request merged**